owl/EASE-OBJ.owl

```diff
@@ -2652,6 +2681,18 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#StructuralDesign -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#StructuralDesign">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Design"/>
+        <rdfs:comment>A design of an object which describes the stability, strength and rigidity of structures and considers the way in which parts of a system or object are arranged. 
+
+The basic objective in structural design is to produce a structure capable of resisting all applied loads without failure during its intended life.</rdfs:comment>
```

Comment:
designs do not produce anything, they describe. When you follow it you may produce something (which is a situation again)

Reply:
Its an objective, there is nowhere in the sentence it is written that way you are interpreting it!

Reply:
"The basic objective in design is to produce" you write.
Probably it is unclear what you mean by "in design"

Reply:
I think its better to remove this sentence, if it is not clear!